### PR TITLE
BD-904:Relicense to Metrological Apache License

### PR DESCRIPTION
**Pull request by ravish-cts**
Relicense OCDM-Widevine to Metrological Apache License header. 
Modified Source and Notice file . 
The License File was up-to-date no change incorporated in it,

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+You may not use the component except in compliance with the License.
+
+The component may include material which is licensed under other licenses / copyrights as
+listed below. Your use of this material within the component is also subject to the terms and
+conditions of these licenses. The LICENSE file contains the text of all the licenses which apply
+within this component.
+
 Copyright (C) 2016 TATA ELXSI
-Copyright (C) 2016 Metrological
+
```
Comment (mhughesacn) [inline NOTICE:11]: Please keep this Metrological copyright.

**Review by mhughesacn — CHANGES_REQUESTED** [review]

File: NOTICE (hunk 1 above)
Comment (pwielders) [reply NOTICE:11]: Done

**Review by pwielders — APPROVED** [review]

**Review by pwielders — APPROVED** [review]

**Comment by mhughesacn** [comment]
Hi Pierre,
I think Ravish left the project so Mallikarjuna did the work under a new pull req (#10, merged) so this should have been abandoned.  (Which is why it was hanging around for so long.)
Thanks,
Martin

**Comment by scthunderbolt** [comment]
Closed due to inactivity, please restore if still needed.

**Pull request closed without merging**